--- meson.build  2023-08-23 18:16:21.481968079 +0000
+++ meson2.build 2023-08-23 18:17:10.789635151 +0000
@@ -209,6 +209,10 @@
 mathlib_dep = cc.find_library('m', required: false)
 pango_deps += mathlib_dep

+# graphite2
+graphite2_dep = cc.find_library('graphite2', required: true)
+pango_deps += graphite2_dep
+
 # gobject
 glib_dep = dependency('glib-2.0', version: glib_req_version)
 gobject_dep = dependency('gobject-2.0', version: glib_req_version)
